p2p/discovery/mdns.go

```diff
@@ -159,7 +164,7 @@ func (m *mdnsService) handleEntry(e *mdns.ServiceEntry) {
 
 	m.lk.Lock()
 	for _, n := range m.notifees {
-		n.HandlePeerFound(pi)
+		go n.HandlePeerFound(pi)
```

Comment:
it looks like _this_ is the only thing this changeset should change. (plus the logs)